src/harvest.h

```diff
@@ -34,6 +34,8 @@ class harvest_list
 
         const harvest_id &id() const;
 
+        const std::string message() const;
```

Comment:
The message is not translated anywhere, nor is it extracted for translation anywhere.

Also: why are you returning a `const` string here?

Reply:
message() return doesn't have to be const when message_ a private member with no setter method but I thought it made the code clearer.